crates/gpui/src/platform/windows/platform.rs

```diff
@@ -684,7 +684,7 @@ impl Platform for WindowsPlatform {
 
     fn read_from_clipboard(&self) -> Option<ClipboardItem> {
         let mut ctx = ClipboardContext::new().unwrap();
-        let content = ctx.get_contents().unwrap();
+        let content = ctx.get_contents().unwrap_or_default();
```

Comment:
Rather than returning a `Some(default)`, we should return a None here. You can do that by replacing this line with:

```rs
let content = ctx.get_contents().ok()?
```

Reply:
Ahhh, I see.